RevenueCatUI/Runtime/Internal/RevenueCatUICallbackHandler.cs

```diff
@@ -0,0 +1,55 @@
+using UnityEngine;
+
+namespace RevenueCat.UI.Internal
+{
+    /// <summary>
+    /// Handles callbacks from native RevenueCat UI platforms
+    /// </summary>
+    internal class RevenueCatUICallbackHandler : MonoBehaviour
+    {
+        private static RevenueCatUICallbackHandler _instance;
+
+#if UNITY_ANDROID && !UNITY_EDITOR
+        private static Platforms.AndroidPaywallPresenter _androidPresenter;
+        private static Platforms.AndroidCustomerCenterPresenter _androidCustomerCenterPresenter;
+#endif
+
+        public static void Initialize()
+        {
+            if (_instance == null)
+            {
+                var go = new GameObject("RevenueCatUI");
+                _instance = go.AddComponent<RevenueCatUICallbackHandler>();
+                DontDestroyOnLoad(go);
+            }
+        }
+
+#if UNITY_ANDROID && !UNITY_EDITOR
+        public static void SetAndroidPresenter(Platforms.AndroidPaywallPresenter presenter)
+        {
+            _androidPresenter = presenter;
+        }
+
+        public static void SetAndroidCustomerCenterPresenter(Platforms.AndroidCustomerCenterPresenter presenter)
```

Comment:
Would it make sense to split responsibilities for paywalls and customer center? I don't think the callbacks should necessarily be tied between both?